src/components/main-view/main-view.jsx

```diff
@@ -1,34 +1,31 @@
-import { useState } from "react";
+import { useState, useEffect } from "react";
 import {MovieCard} from "../movie-card/movie-card";
 import {MovieView} from "../movie-view/movie-view"
 import { title } from "process";
 
 
 export const MainView = () => {
-    const [movies, setMovies] = useState([
-        {
-            id: 1,
-            title: "Focus",
-            image: "focus.png",
-
-            director: "Glenn Ficarra & John Requa"
-        },
-
-        {
-            id: 2,
-            title: "Zodiac",
-            image: "zodiac.png",
-            director: "David Fincher"
-        },
-        {
-            id: 3,
-            title: "Now You See Me",
-            image: "nowyouseeme.png",
-            director: "Louis Leterrier"
-        }
-    ]);
+    const [movies, setMovies] = useState([]);
 
     const [selectedMovie, setSelectedMovie] = useState(null);
```

Comment:
I got an error in my browser log (always check that for possible errors if you test your app), the error was because  you commented out this line of code [10], was wondering why you did that.